[2024-11-04T01:13:55.572+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_feature_select_and_data_augmentation.feature_selection_task manual__2024-11-04T01:13:54.610213+00:00 [queued]>
[2024-11-04T01:13:55.577+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: DAG_feature_select_and_data_augmentation.feature_selection_task manual__2024-11-04T01:13:54.610213+00:00 [queued]>
[2024-11-04T01:13:55.578+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-11-04T01:13:55.578+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-11-04T01:13:55.578+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-11-04T01:13:55.585+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): feature_selection_task> on 2024-11-04 01:13:54.610213+00:00
[2024-11-04T01:13:55.590+0000] {standard_task_runner.py:55} INFO - Started process 290 to run task
[2024-11-04T01:13:55.592+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'DAG_feature_select_and_data_augmentation', 'feature_selection_task', 'manual__2024-11-04T01:13:54.610213+00:00', '--job-id', '219', '--raw', '--subdir', 'DAGS_FOLDER/feature_and_augm_dag.py', '--cfg-path', '/tmp/tmp4ttu08j2']
[2024-11-04T01:13:55.594+0000] {standard_task_runner.py:83} INFO - Job 219: Subtask feature_selection_task
[2024-11-04T01:13:55.628+0000] {task_command.py:388} INFO - Running <TaskInstance: DAG_feature_select_and_data_augmentation.feature_selection_task manual__2024-11-04T01:13:54.610213+00:00 [running]> on host b77d411e2e76
[2024-11-04T01:13:55.659+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=House_Price_Prediction Team
AIRFLOW_CTX_DAG_ID=DAG_feature_select_and_data_augmentation
AIRFLOW_CTX_TASK_ID=feature_selection_task
AIRFLOW_CTX_EXECUTION_DATE=2024-11-04T01:13:54.610213+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-11-04T01:13:54.610213+00:00
[2024-11-04T01:13:55.664+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/feature_and_augm_dag.py", line 62, in feature_selection_callable
    raise ValueError("No encoded data found in XCom for 'encoded_data'")
ValueError: No encoded data found in XCom for 'encoded_data'
[2024-11-04T01:13:55.668+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=DAG_feature_select_and_data_augmentation, task_id=feature_selection_task, execution_date=20241104T011354, start_date=20241104T011355, end_date=20241104T011355
[2024-11-04T01:13:55.673+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 219 for task feature_selection_task (No encoded data found in XCom for 'encoded_data'; 290)
[2024-11-04T01:13:55.690+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-11-04T01:13:55.703+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
